# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../csync/csync-cmakelists.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- csync-0.44.0/cmake/Modules/FindLibsmbclient.cmake.orig	2010-10-30 13:10:28.000000000 -0230
+++ csync-0.44.0/cmake/Modules/FindLibsmbclient.cmake	2010-10-30 13:09:41.000000000 -0230
@@ -25,16 +25,19 @@
       /usr/include
       /usr/local/include
       /opt/local/include
+      /opt/samba/include
       /sw/include
   )
 
   find_library(SMBCLIENT_LIBRARY
     NAMES
       smbclient
+      libsmbclient.*
     PATHS
       /usr/lib
       /usr/local/lib
       /opt/local/lib
+      /opt/samba/lib
       /sw/lib
   )
 
